automatic/zotero/zotero.nuspec

```diff
@@ -2,30 +2,32 @@
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
+    <!-- == PACKAGE SPECIFIC SECTION == -->
     <id>zotero</id>
-    <title>Zotero</title>
+    <version>7.0.7</version>
```

Comment:
Do not update the version number. Please revert it back to `7.0.5`

Reply:
Sure.